packages/web-runtime/src/store/user.js

```diff
@@ -31,18 +31,26 @@ const actions = {
     // clear dynamic navItems
     context.dispatch('clearDynamicNavItems')
 
-    // clear oidc client state
-    vueAuthInstance.clearLoginState()
+    if (options.clearOIDCLoginState) {
+      // clear oidc client state
+      vueAuthInstance.clearLoginState()
+    }
```

Comment:
@kulmann this code path triggers a redirect already and interferes with redirecting to the oc10 log out,
I don't call ` vueAuthInstance.clearLoginState()`anymore if we are logged in via oauth2, is this ok?

Reply:
Is it really sufficient for oauth2 to skip that step? o_O